app/views/reports/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="container">
-  <h1 class="dashboard-title-report"><%= @project.title %> Report</h1>
+  <h1 class="dashboard-title-report"><%= @project.title %> Report <br> <span class="status-badge locked <%= "hide" if @project.locked_at.nil? %>">Locked</span> </h1>
```

Comment:
We can make the markup a bit more semantic, but moving the badge out of the h1 tag.
```suggestion
  <h1 class="dashboard-title-report"><%= @project.title %> Report </h1>
  <div><span class="status-badge locked <%= "hide" if @project.locked_at.nil? %>">Locked</span></div>
```